addons/binding/org.openhab.binding.chromecast/README.md

```diff
@@ -30,7 +30,35 @@ Chromecast devices can also be manually added. The only configuration parameter
 |-------------|--------|-----------------------------|
 | control | Player | Player control; currently only supports play/pause and does not correctly update, if the state changes on the device itself |
 | volume | Dimmer | Control the volume, this is also updated if the volume is changed by another app |
+| mute | Switch | Can mute the chromecast, updated if changed from another app |
 | playuri | String | Can be used to tell the Chromecast to play media from a given url |
+| appName | String | The current app's name |
+| appId | String | The current app's id |
+| idling | Switch | Whether the app is idling or playing |
+| currentTime | Number | Current time of the playing media |
+| duration | Number | How long the media is |
+| metadataType | String | The type of the currently playing media. One of GENERIC, MOVIE, TV_SHOW, AUDIO_TRACK, PHOTO |
+| albumArtist | String | Reported information about the currently playing media |
+| albumName | String |Reported information about the currently playing media|
+| artist | String | Reported information about the currently playing media |
+| broadcastDate | Number |Reported information about the currently playing media|
+| composer | String | Reported information about the currently playing media |
+| creationDate | Number | Reported information about the currently playing media |
+| discNumber | Number |Reported information about the currently playing media|
+| episodeNumber | Number |Reported information about the currently playing media|
+| height | Number | Reported information about the currently playing media |
+| images | String | Reported information about the currently playing media |
+| locationName | String | Reported information about the currently playing media |
```

Comment:
I have no idea. These keys are supplied by Google. I just figured it's better to make them all available than exclude the ones I didn't like.